docs/reference/semaphore-changelog.md

```diff
@@ -9,10 +9,10 @@ We continuously deploy changes that improve the product for you.
 This page is updated on a weekly basis.
 ### Week of August 17, 2020
 **(New) Install-package command**  
-"Toolbox" command line tools were extended to include `install-package` utility.
-The `install-package` command help with package installations by automatically 
+"Toolbox" command line tools have been extended to include `install-package` utility.
+The `install-package` command can help with package installations by automatically 
 caching the desired packages and their dependencies.  
-To find out more please check [install-package](https://docs.semaphoreci.com/reference/toolbox-reference/#install-package) documentation page.
+To find out more please check [install-package](https://docs.semaphoreci.com/reference/toolbox-reference/#install-package), documentation page.
```

Comment:
comma is not in the right place, it should be:
To find out more, please check...